de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_Server_Manual.adoc

```diff
@@ -0,0 +1,236 @@
+// The connection can be secured via HTTPS.
+Every user can only access certain provided Server Repositories.
+
+The connected SVN/git repositories can also be directly changed from _Clients using the Virtual Satellite UI_.
+The server resolves these conflicts and also handles simultaneous changes from multiple REST clients.
+The SVN/git repository is supposed to be the single source of truth in those cases.
+
+=== Components
+
+Figure <<RestServerComponents>> gives a more detailed view on the REST Server components.
+
+.Components of the Headless REST Server.
+[#RestServerComponents]
+image::chapterServerComponents/REST_server_components.png[Server Components]
+
+The server is divided into three main layers: the _REST API_ (View), _Server Controller_ and _Persistence_ (Model).
+Between the API and the controller also is an _Access check_ layer.
+
+A REST Client can send a request to the provided _REST API_ that is partitioned into multiple Resources:
+An _Admin Resource_, in which an administrator can manipulate the Server Configuration (_ServerConf_), e.g. add Repository Configurations (_RepoConf_),
+and a _Repository Resource_. Each _Server Repository_ provided has its own _Subresource_.
+
+In the _Access check_ layer the credentials are obtained from the HTTP Header and it is checked if the requested Resources can be accessed.
+The credentials are handed over to be used for user role handling in the Virtual Satellite _Data Model_.
+
+The _Server Controller_ handles requests to the _Admin Resource_. In its _ServerConf_ the server settings are stored.
+Requests to a specific _Repository Subresource_ via the _Repository Resource_ directly change the underlying _Data Model_.
+
+The persistence level is composed of _Server Repositories_, each with a _Repository Configuration_ and a _Data Model_. 
+The _Data Model_ is connected to a _SVN_ or _git_ backend.
+
+== The REST API
+
+The REST API can be accessed via the 
+[subs=+quotes]
+....
+{*server adress*}:8000/rest
+....
+and currently supports multiple APIs for different purposes.
+Each API can be accessed via it's *name* and *version number*:
+[subs=+quotes]
+....
+{*server adress*}:8000/rest/{*name*}/{*version number*}
+....
+
+Currently the following APIs are provided:
+
+* The Model API to access Virtual Satellite Models ({*name*} = *model*)
+* The Management API for administrative tasks ({*name*} = *management*)
+
+The following documentation of the REST endpoints is automatically generated using Swagger.
+For each API the swagger documentation also is provided via the REST API at:
+[subs=+quotes]
+....
+{*server adress*}:8000/rest/{*name*}/{*version number*}/swagger.json or /swagger.yaml
+....
+
+It can help you, if you want to connect Virtual Satellite to your own application.
+E.g. it can be used to automatically generate your client, using Swaggers Codegen capabilities.
+
+#TODO: insert generated swagger documentation for the APIs here#
```

Comment:
Are you going to handle this TODO in this PR?

Reply:
No, I want to adress it in https://github.com/virtualsatellite/VirtualSatellite4-Core/issues/834. But before that I would like to merge the two open swagger PRs.